samcli/commands/sync/command.py

```diff
@@ -197,6 +197,9 @@ def do_cli(
     click.echo("\t\tA different default S3 bucket can be set in samconfig.toml")
     click.echo("\t\tOr by specifying --s3-bucket explicitly.")
 
+    click.echo("\n\t\tDefault capabilities applied: CAPABILITY_NAMED_IAM, CAPABILITY_AUTO_EXPAND")
```

Comment:
nit: we can move default capabilities into a static var and use it here and at line 99, so that we don't need to change 2 places in the future.